src/main/target/NAZE/target.h

```diff
@@ -157,7 +157,9 @@
 // USART2, PA3
 #define BIND_PIN                PA3
 
+#if !defined(BEEBRAIN)
```

Comment:
Only beebrain is a naze equialant here what uses brushed .

There is afromini, naze and beebrain from this target